pkg/parser/model/ddl.go

```diff
@@ -448,6 +547,12 @@ type Job struct {
 
 	// TraceInfo indicates the information for SQL tracing
 	TraceInfo *TraceInfo `json:"trace_info"`
+
+	// BDRRole indicates the role of BDR cluste when executing this DDL.
+	BDRRole string `json:"bdr_role"`
+
+	// CDCWriteSource indicates the source of CDC write.
+	CDCWriteSource uint64 `json:"cdc_write_source"`
```

Comment:
Maybe add "omitempty" to these two json tags?

Reply:
What is your concern?

Reply:
OK. This is only helpful when BDRRole is zero in most cases.